README.ko.md

```diff
@@ -0,0 +1,73 @@
+# @daldalso/i18n
+React.js를 위한 국제화(i18n) 라이브러리
+
+<p style="text-align: center;"><a href="README.md">English</a> | 한국어</p>
```

Comment:
This won't work in GitHub. You have to use `<p align="center">` instead.

Reply:
[e88ddfa](https://github.com/JJoriping/i18n/pull/2/commits/e88ddfabe7541e562d82cf6b7fc37babd78b3d32)